pydicom/encaps.py

```diff
@@ -69,6 +69,8 @@ def defragment_data(data):
 def read_item(fp):
     """Read and return a single Item in the
     fragmented data stream"""
+
+    logger = pydicom.config.logger
```

Comment:
I find it a bit nasty. But ok. Can you open an issue so that we can get back to this circular dependency. Maybe we can figure a nicer manner to do this.